modules/rosa-sharing-vpc-dns-and-roles.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * networking/rosa-shared-vpc-config.adoc
+:_content-type: PROCEDURE
+[id="rosa-sharing-vpc-dns-and-roles_{context}"]
+= Creating your DNS and cluster creation roles
+
+After the VPC-owning AWS account user creates a virtual private cloud, subnets, and an IAM role for sharing the VPC resources, the cluster-creating AWS account user must create a DNS domain and Operator roles to share with the VPC-owning AWS account.
+
+.Prerequisites
+
+* You have the ARN for the IAM role that is used to share your VPC.
+
+.Procedure
+
+. On the cluster creator's AWS account, you create a DNS domain with the following command:
++
+[source,terminal]
+----
+$ rosa create dns-domain
+----
++
+The command creates a generated DNS domain.
++
+[source,terminal]
+----
+I: DNS domain '14eo.p1.openshiftapps.com' has been created.
+I: To view all DNS domains, run 'rosa list dns-domains
+----
+. After creating the DNS domain, the cluster creator needs to create an OIDC configuration. The following command produces the OIDC config ID that you need:
++
+[source,terminal]
+----
+$ rosa create oidc-config
+----
+. After the OIDC configuration is created, you create the Operator roles using the following command:
++
+[source,terminal]
+----
+$ rosa create operator-roles --oidc-config-id <oidc-config-ID> <1>
+    --installer-role-arn <Installer_Role> <2>
+    --shared-vpc-role-arn <Created_VPC_Role_Arn> <3>
+    --prefix <operator-prefix> <4>
+----
+--
+<1> Provide the ODIC configuration ID that you created in the previous step.
+<2> Provide the cluster creator's installer ARN that was created as part of the `rosa create account-roles` process.
+<3> Provide the ARN for the role the VPC owner created.
+<4> Create a prefix for the Operator roles, noting that for sharing a VPC, when you create your cluster, you must use this prefix as the cluster name.
+--
+
+When complete, share the created DNS domain, your Ingress Operator Cloud Credentials and Installer roles' ARN, and the Operator role prefix with the VPC-owning AWS account user.
```

Comment:
@EricPonvelle As this is very important configure steps for Shared-VPC install, can we make it more clear in the doc?

@oriAdler WDYT?